rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.adoc

```diff
@@ -1,8 +1,11 @@
 :_mod-docs-content-type: ASSEMBLY
-include::_attributes/attributes-openshift-dedicated.adoc[]
 [id="rosa-nodes-about-autoscaling-nodes"]
 = About autoscaling nodes on a cluster
+
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
 :context: rosa-nodes-about-autoscaling-nodes
+
```

Comment:
Adding spaces to the headers _removes_ the topic map. These line breaks should be removed